test_scripts/API/VehicleData/Refactor_Fuel_Information_Related_Vehicle_Data/004_OnVehicleData_with_new_fuelRange_structure.lua

```diff
@@ -0,0 +1,47 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal:https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0256-Refactor-Fuel-Information-Related-Vehicle-Data.md
+-- Description: SDL successfully processes a valid OnVehicleData notification and transfers
+-- it to a mobile app with all new FuelRange structure
+-- In case:
+-- 1) App is subscribed to get `FuelRange` data
+-- 2) HMI sends valid  OnVehicleData notification with all parameters of structure `FuelRange`
+-- SDL does:
+-- 1) process this notification and transfer it to mobile
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require('test_scripts/API/VehicleData/Refactor_Fuel_Information_Related_Vehicle_Data/common')
+
+-- [[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+local params = {
```

Comment:
According to comments above please move the structure to common file and remove it from the script.